app/controllers/attributes_controller.rb

```diff
@@ -29,16 +35,48 @@ def show
   def update
     attributes = params.fetch(:attributes).permit!.to_h
 
-    oauth_response = OidcClient.new.bulk_set_attributes(
-      attributes: attributes,
-      access_token: @govuk_account_session[:access_token],
-      refresh_token: @govuk_account_session[:refresh_token],
-    )
+    return unless validate_attributes(attributes.keys)
+
+    access_token = @govuk_account_session[:access_token]
+    refresh_token = @govuk_account_session[:refresh_token]
+
+    local_attributes = attributes.select { |name, _| user_attributes.stored_locally? name }
```

Comment:
Why do you need the second `_` param here?

Reply:
Aha